packages/hardware-trezor/src/transformers/tx.ts

```diff
@@ -0,0 +1,54 @@
+import * as Trezor from 'trezor-connect';
+import { Cardano } from '@cardano-sdk/core';
+import { TrezorTxTransformerContext } from '../types';
+import { util as deprecatedUtil } from '@cardano-sdk/key-management';
+import { mapAuxiliaryData, mapCerts, mapTxIns, mapTxOuts, mapWithdrawals } from './';
+import { mapTokenMap } from './assets';
+
+/**
+ * Temporary transformer function that returns a partial
+ * trezor.CardanoSignTransaction object which can be merged
+ * into the extisting implementation (later we should refactor
+ * this function to the Transformer interface like in the
+ * hardware-ledger package)
+ */
+const trezorTxTransformer = async (
+  body: Cardano.TxBody,
+  context: TrezorTxTransformerContext
+): Promise<Partial<Trezor.CardanoSignTransaction>> => ({
```

Comment:
We probably should be able to remove this `Partial<T>` bit at this stage (now that all mappers are in place)

Reply:
There is still one missing: https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/key-management/src/util/mapHardwareSigningData.ts#L987